backend/server/src/db/mod.rs

```diff
@@ -255,4 +255,7 @@ struct Migration {
 
 static MIGRATIONS: Lazy<BTreeMap<u64, Migration>> = include_migrations![
     1: "realms",
+    2: "xtea",
+    3: "keys",
+    4: "events",
```

Comment:
Given that we might want to use xtea stuff in `realms`, too, and because `xtea` and `keys` seem to be more of a meta-setup, I would put xtea and keys before realms.

Reply:
Yeah, I think I agree with that. It's done. :)